ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/B11rSubCommand.java

```diff
@@ -162,6 +195,11 @@ public void run() {
           config.set("txs", t8nReader.readTree(reader));
         }
       }
+      if (!withdrawals.equals(stdinPath)) {
+        try (FileReader reader = new FileReader(withdrawals.toFile(), StandardCharsets.UTF_8)) {
+          config.set("withdrawals", t8nReader.readTree(reader));
```

Comment:
I assume using `t8nReader` as a name for the ObjectMapper here is just copypasta?

Reply:
The irony being this was to remove copypasta.  Will fix.